server/service/osquery_utils/queries.go

```diff
@@ -754,6 +760,46 @@ func directIngestBattery(ctx context.Context, logger log.Logger, host *fleet.Hos
 	return ds.ReplaceHostBatteries(ctx, host.ID, mapping)
 }
 
+func directIngestWindowsUpdateHistory(
+	ctx context.Context,
+	logger log.Logger,
+	host *fleet.Host,
+	ds fleet.Datastore,
+	rows []map[string]string,
+	failed bool,
+) error {
+	if failed {
+		level.Error(logger).Log("op", "directIngestWindowsUpdateHistory", "err", "failed")
+		return nil
+	}
+
+	// The windows update history table will also contain entries for the Defender Antivirus. Unfortunately
+	// there's no reliable way to differentiate between those entries and Cumulative OS updates.
+	// Since each antivirus update will have the same KB ID, but different 'dates', to
+	// avoid trying to insert duplicated data, we group by KB ID and then take the most 'out of
+	// date' update in each group.
```

Comment:
Should we open a osquery issue for this?

Reply:
I think osquery is doing the right thing here - Windows Defender updates are shown under Windows updates, so that's the expected behavior.

Reply:
> there's no reliable way to differentiate between those entries and Cumulative OS updates

Meant an issue for this part ^

Reply:
Here's what I meant by 'Windows Defender updates are shown under Windows updates':

![image](https://user-images.githubusercontent.com/8041318/184026544-55b629c0-9926-4227-a14a-e841d889e54d.png)

So technically, we could say osquery is doing the right thing here. 

We could filter those entries by doing something like ` ... AND title NOT LIKE '%Microsoft Defender Antivirus%' ...` but I'm not sure that's a good idea (we might skip some valid entries).